src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -177,11 +180,29 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
                   />
                 </Flex>
 
-                <Separator my={1} />
+                <Separator my={2} />
 
                 <PriceRangeFilter />
 
                 <Separator my={2} />
+
+                <Text variant="sm-display">
+                  Tell us more about what you’re looking for
+                </Text>
+                <Spacer y={1} />
+                <TextArea
+                  name="details"
+                  placeholder="For example, a specific request such as ‘spin paintings series’ or ‘signed prints’"
+                  onChange={({ value }) => {
+                    setFieldValue("details", value)
+                  }}
+                  onBlur={handleBlur}
+                  value={values.details}
+                  error={errors.details}
+                  maxLength={700}
```

Comment:
**suggestion**: How does this prop differ from `characterLimit` in [the Palette docs](https://palette.artsy.net/elements/textarea/#with-character-limit)? Either way, we might want to differ this question of character limit for another PR. I dug around today and the inquiry flow, which has a similar text area input, currently allows quite a bit of input from the collector.

Reply:
It was just my feeling that 200 characters might be not enough. I've used this [dummy text generator](https://www.blindtextgenerator.com/lorem-ipsum) to visualise text area with different lengths and ended up with 700 🤷